index.js

```diff
@@ -1,12 +1,29 @@
-function destroyVue() {
-  this.$destroy();
-  document.removeEventListener('turbolinks:before-cache', destroyVue.bind(this))
+// TL means Turbolinks
+function isTLCacheDisabled() {
```

Comment:
Same with this one. Let's err on the side of verbosity for readability. If you can think of a different method name, that works too.